pkg/test/framework/label/labels.go

```diff
@@ -24,12 +24,17 @@ const (
 	// IPv4 indicates a test is only compatible with IPv4 clusters.
 	// Any usage of this should have an associated GitHub issue to make it compatible with IPv6
 	IPv4 Instance = "ipv4"
+
+	// TODO: Make sure that we need this label
+	// Multicluster indicates that the test requires a multicluster configuration.
+	Multicluster Instance = "multicluster"
```

Comment:
I don't think we'll need this

Reply:
Fixed in https://github.com/maistra/istio/commit/29a3ba5533e0c01c0bd2280190239de6dd8f9249